webapp/public/release.manifest

```diff
@@ -16,6 +16,7 @@ CACHE:
 /blb/pxteditor.js
 /blb/pxtsim.js
 /blb/pxtembed.js
+/blb/mainAsset.js
```

Comment:
mainAsset is too generic, it should be assetEditor.js or pxtasseteditor.js (all lowercase)

Reply:
The manifest is for the pxt editor, I don’t think you need this here since the asset editor is already built into Pxt